build/Projects/OpenTelemetry.Instrumentation.AspNet.proj

```diff
@@ -14,7 +14,8 @@
     <PackProjects Include="$(RepoRoot)\src\OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule\OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule.csproj" />
     <PackProjects Include="$(RepoRoot)\src\OpenTelemetry.Instrumentation.AspNet\OpenTelemetry.Instrumentation.AspNet.csproj" />
 
-    <TestProjects Include="$(RepoRoot)\test\OpenTelemetry.Instrumentation.AspNet*.Tests\*.Tests.csproj" />
```

Comment:
FYI the wildcard here `OpenTelemetry.Instrumentation.AspNet*` was causing `AspNetCore` tests to be included which weren't part of the build so test run were failing for AspNet proj. Took me a while to figure that out 🤣